openexr-2.2.0 fails with gcc-9 (std=c++17)

```
In file included from ImathShear.h:49,
                 from ImathShear.cpp:44:
ImathVec.h:228:41: error: ISO C++17 does not allow dynamic exception specifications
  228 |     const Vec2 &        normalizeExc () throw (IEX_NAMESPACE::MathExc);
      |                                         ^~~~~
ImathVec.h:232:48: error: ISO C++17 does not allow dynamic exception specifications
  232 |     Vec2<T>             normalizedExc () const throw (IEX_NAMESPACE::MathExc);
      |                                                ^~~~~
ImathVec.h:440:41: error: ISO C++17 does not allow dynamic exception specifications
  440 |     const Vec3 &        normalizeExc () throw (IEX_NAMESPACE::MathExc);
      |                                         ^~~~~
ImathVec.h:444:48: error: ISO C++17 does not allow dynamic exception specifications
  444 |     Vec3<T>             normalizedExc () const throw (IEX_NAMESPACE::MathExc);
      |                                                ^~~~~
ImathVec.h:622:37: error: ISO C++17 does not allow dynamic exception specifications
  622 |     const Vec4 &    normalizeExc () throw (IEX_NAMESPACE::MathExc);
```

As the workaround it needs `-std=gnu++0x` as the test program already has. I will create PR.

--- a/src/openexr.mk
+++ b/src/openexr.mk
@@ -32,6 +32,7 @@ define $(PKG)_BUILD
         --prefix='$(1)/ilmbase' \
         --enable-threading=no \
         --disable-posix-sem \
+        CXXFLAGS="-g -O2 -std=gnu++0x" \
         CONFIG_SHELL=$(SHELL) \
         SHELL=$(SHELL)
     $(MAKE) -C '$(1)/$(ilmbase_SUBDIR)' -j '$(JOBS)' install \
@@ -42,7 +43,7 @@ define $(PKG)_BUILD
         --disable-posix-sem \
         --disable-ilmbasetest \
         PKG_CONFIG='$(PREFIX)/bin/$(TARGET)-pkg-config' \
-        CXXFLAGS="-g -O2"
+        CXXFLAGS="-g -O2 -std=gnu++0x"
     # build the code generator manually
     cd '$(1)/IlmImf' && $(BUILD_CXX) -O2 \
         -I'$(1)/ilmbase/include/OpenEXR' \

